src/util/algoliaUtils.js

```diff
@@ -7,7 +7,7 @@ export const fundraisingIndices = [
 	{ value: 'dev_fundraising_amount_asc', label: 'Amount: low to high' },
 	{ value: 'dev_fundraising_amount_desc', label: 'Amount: high to low' },
 	{ value: 'dev_fundraising_amount_remaining', label: 'Amount left' },
-	{ value: 'dev_fundraising_expiring_soon', label: 'Expiring soon' },
+	{ value: 'dev_fundraising_expiring_soon', label: 'Ending soon' },
```

Comment:
Good for now. Obvs this has a long tail of other work to actually remove the "expiring" text from functional Algolia portions of the site.